src/transaction.rs

```diff
@@ -80,8 +80,9 @@ impl Decodable for OutPoint {
     }
 }
 
-impl BitcoinHash<Txid> for OutPoint {
-    fn bitcoin_hash(&self) -> Txid {
+impl OutPoint {
+    /// Remove this
+    pub fn hash(&self) -> Txid {
```

Comment:
Yeah this should probably be inlined where-ever this is used because it's strange.